srcpkgs/Venom/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'Venom'
+pkgname=Venom
+version=0.5.4
+revision=1
+build_style=meson
+hostmakedepends="vala pkg-config"
+makedepends="gtk+3-devel libgee-devel gspell-devel
+ libcanberra-devel sqlcipher-devel json-glib-devel
+ libsoup-devel toxcore-devel libsodium-devel
+ opus-devel libvpx-devel gstreamer1-devel"
+short_desc="Modern Tox client for the desktop"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="://github.com/naxuroqa/Venom/"
+distfiles="https://github.com/naxuroqa/Venom/archive/${version}.tar.gz"
+checksum=2e4bbab0b68a1a38ffcc5f9991fb5d95391c207e931edb2ce6a1e1edf6795b1b
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" glib-devel"
```

Comment:
just add this to `hostmakedepends`, no conditional.

Reply:
Done